packages/node_modules/@webex/internal-plugin-mercury/src/socket/socket-base.js

```diff
@@ -193,7 +195,7 @@ export default class Socket extends EventEmitter {
       const WebSocket = Socket.getWebSocketConstructor();
 
       this.logger.info('socket: creating WebSocket');
-      const socket = new WebSocket(url);
+      const socket = new WebSocket(url, [], options);
```

Comment:
What was options used for before? i.e. See you did not add it.

Reply:
It was used above with the Reflect.defineProperty to see the options on the socket object, but this does not appear to propagate them to the ws, since we could not create the web socket via the proxy without this change.

Reply:
Yes, I'd assume that was always the intention

Reply:
It does look like you "fixed the glitch" here. Let's hope no one was counting on the bug!

![img](https://www.autodesk.com/products/fusion-360/blog/wp-content/uploads/2019/06/dd4f2296-2186-45d6-b42a-10966b4f9263_text.gif)

Reply:
Yeah, my bad on this one. Although I did try reverting this back to passing just the url and the Jenkins failures still happened. Anyway, at this point I'm just happy it's passing!